### ISSUE TITLE
https://github.com/jupyter/jupyter_events needs maintenance.

### ISSUE THREAD

**Issue by Carreau** [jupyter-governance/ec-team-compass#84]
It has been breaking downstream projects (including jupyter_server CI) for some time due to deprecation and various other issues. 

I'll try to spend a few hours on it as I can; but it will need a long term maintenance plan.

**Comment by Carreau** [comment]
I fixed the critical things and published a new release because it was critical and I was able to afford the few hours; but having regular monitoring of issues/PRs on this package and other woudl be a good thing.

**Comment by jasongrout** [comment]
Thanks @Carreau! What subproject council do you think is responsible for this repo?

**Comment by Carreau** [comment]
I don't know. The only visible maintainer is @Zsailer ; as AFAICT from the PR it affect jupyter-server and jupyter-lab. 

I did a 0.11 release to fix some failing CI in jupyter-server, but I might yank it as it seem to have deep reaching consequences. 

In general I think there should be tools to check the mapping activity of GH-repos, to pypi packages, and a file that describe who should be responsible to check and maybe auto-open issues here if a repo seem unmaintained.

**Comment by afshin** [comment]
I would propose this project be part of Jupyter Server.

**Comment by Zsailer** [comment]
> I would propose this project be part of Jupyter Server.

I agree. I will trigger the transfer by opening a vote with the team. 

> I fixed the critical things and published a new release because it was critical and I was able to afford the few hours; but having regular monitoring of issues/PRs on this package and other woudl be a good thing.

Thank you, Matthias. I appreciate your time here. I'm in the same boat—doing my best to contribute/maintain this project when I get space/time. We do track this in the Jupyter Server meeting, but it's fallen through the cracks.

**Comment by Carreau** [comment]
for what is it worth, jupyter_events needs a release to fix some downstream issues, but I will not make it. 

Closing as I'm unlikey to spend any time on this.

**Comment by Zsailer** [comment]
Thanks, @Carreau. I went ahead and cut a release. I'll monitor the next few days in case any new bugs are reported.
